dancepose/visualize_errors_on_video.py

```
@@ -0,0 +1,315 @@
+Визуализация ошибок на видео
+Наложение меток на существующее overlay видео
+"""
+
+import cv2
+import json
+import numpy as np
+from pathlib import Path
+import sys
+
+
+def draw_error_markers(frame, errors_at_frame, frame_height, scale=1.0):
```

Ruff (0.14.5)

Unused function argument: `frame_height`

(ARG001)